tests/security/apparmor/aa_status.pm

```diff
@@ -17,21 +17,35 @@
 # aa-status.
 # - Check if apparmor is active
 # - Run aa-status, check the output for strings about modules/profiles/processes
+#   and verify "aa-status" can "handle profile with name contains '('".
 # and strings enforced, complain, unconfined and loaded.
 # Maintainer: llzhao <llzhao@suse.com>
-# Tags: poo#36874, poo#44912
+# Tags: tc#1767574, poo#81727, poo#36874, poo#44912
```

Comment:
not a big deal, but why tc# is the first order here :)

Reply:
> not a big deal, but why tc# is the first order here :)

No reason just for "looks nice".

Reply:
ok I see.